src/core/library.c

```diff
@@ -276,6 +276,14 @@ MsQuicLibraryUninitialize(
         MsQuicLib.UnregisteredSession = NULL;
     }
 
+    //
+    // Ensure all datapath workers are shut down, so new connections won't be 
+    // created while we shut down worker threads.
+    //
+    if (MsQuicLib.Datapath != NULL) {
```

Comment:
I don't believe `MsQuicLib.Datapath` should ever be NULL. You can remove the check.